<ui version="4.0" >
 <class>ICQOrgAffInfoWidget</class>
 <widget class="QWidget" name="ICQOrgAffInfoWidget" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>394</width>
    <height>333</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" >
   <property name="margin" >
    <number>9</number>
   </property>
   <property name="spacing" >
    <number>6</number>
   </property>
   <item>
    <widget class="QGroupBox" name="groupBox" >
     <property name="title" >
      <string>Organization Type</string>
     </property>
     <layout class="QGridLayout" >
      <property name="margin" >
       <number>9</number>
      </property>
      <property name="spacing" >
       <number>6</number>
      </property>
      <item row="1" column="0" >
       <widget class="QLineEdit" name="org2CategoryEdit" />
      </item>
      <item row="2" column="0" >
       <widget class="QLineEdit" name="org3CategoryEdit" />
      </item>
      <item row="1" column="1" >
       <widget class="QLineEdit" name="org2KeywordEdit" />
      </item>
      <item row="2" column="1" >
       <widget class="QLineEdit" name="org3KeywordEdit" />
      </item>
      <item row="0" column="0" >
       <widget class="QLineEdit" name="org1CategoryEdit" />
      </item>
      <item row="0" column="1" >
       <widget class="QLineEdit" name="org1KeywordEdit" />
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_2" >
     <property name="title" >
      <string>Past Affiliation</string>
     </property>
     <layout class="QGridLayout" >
      <property name="margin" >
       <number>9</number>
      </property>
      <property name="spacing" >
       <number>6</number>
      </property>
      <item row="0" column="0" >
       <widget class="QLineEdit" name="aff1CategoryEdit" />
      </item>
      <item row="0" column="1" >
       <widget class="QLineEdit" name="aff1KeywordEdit" />
      </item>
      <item row="1" column="0" >
       <widget class="QLineEdit" name="aff2CategoryEdit" />
      </item>
      <item row="2" column="0" >
       <widget class="QLineEdit" name="aff3CategoryEdit" />
      </item>
      <item row="2" column="1" >
       <widget class="QLineEdit" name="aff3KeywordEdit" />
      </item>
      <item row="1" column="1" >
       <widget class="QLineEdit" name="aff2KeywordEdit" />
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer>
     <property name="orientation" >
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" >
      <size>
       <width>241</width>
       <height>16</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>org1CategoryEdit</tabstop>
  <tabstop>org1KeywordEdit</tabstop>
  <tabstop>org2CategoryEdit</tabstop>
  <tabstop>org2KeywordEdit</tabstop>
  <tabstop>org3CategoryEdit</tabstop>
  <tabstop>org3KeywordEdit</tabstop>
  <tabstop>aff1CategoryEdit</tabstop>
  <tabstop>aff1KeywordEdit</tabstop>
  <tabstop>aff2CategoryEdit</tabstop>
  <tabstop>aff2KeywordEdit</tabstop>
  <tabstop>aff3CategoryEdit</tabstop>
  <tabstop>aff3KeywordEdit</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
